src/components/molecules/GameCard/GameCard.js

```diff
@@ -13,21 +14,61 @@ export function GameCard({
     gameTitle = 'Omega Long Game Title',
     releaseDate = '12/22/22',
 }) {
+    const { ratingStyle, sideBarStyle } = useMemo(() => {
+        if (typeof rating !== 'number')
+            return {
+                ratingStyle: styles.noRating,
+                sideBarStyle: styles.sideBarNoRating,
+            }
+
+        switch (true) {
+            case rating >= 75:
+                return {
+                    ratingStyle: styles.highRating,
+                    sideBarStyle: styles.sideBarHigh,
+                }
+            case rating >= 50:
+                return {
+                    ratingStyle: styles.midRating,
+                    sideBarStyle: styles.sideBarMid,
+                }
+            default:
+                return {
+                    ratingStyle: styles.lowRating,
+                    sideBarStyle: styles.sideBarLow,
+                }
+        }
+    }, [rating])
+    const ratingText = useMemo(
+        () => (typeof rating === 'number' ? Math.round(rating) : rating),
+        [rating]
+    )
+
+    const SUMMARY_LENGTH = 275
+
     return (
-        <div data-testid="game-card" className={styles.gameCard}>
-            <img
-                src={constructImgURL(imageId)}
-                alt={`Cover art for ${gameTitle}`}
-                className={styles.gameCardCoverImg}
-            />
+        <div
+            data-testid="game-card"
+            className={`${styles.gameCard} ${sideBarStyle}`}
+        >
+            <div className={styles.gameImgContainer}>
+                <img
+                    src={constructImgURL(imageId)}
+                    alt={`Cover art for ${gameTitle}`}
+                />
+            </div>
             <div className={styles.gameCardHead}>
-                <Rating value={rating} />
-                <div>
+                <Rating ratingText={ratingText} bgColor={ratingStyle} />
+                <div className={styles.headText}>
                     <h2>{gameTitle}</h2>
                     <p>{releaseDate}</p>
                 </div>
             </div>
-            <p className={styles.summary}>{summary}</p>
+            <p className={styles.summary}>
+                {summary.length >= SUMMARY_LENGTH
```

Comment:
I would say to move this logic up out of the return function. It could probably also stand to be `memo`d.